<launch>
  <arg name="calibration_file" default="$(find visual_servoing)/calibration_data/ost.yaml"/>
  <arg name="camera" default="/t265/stereo_ir/left/fisheye_image_raw"/>
 
  <param name="calibration_file" value="$(arg calibration_file)" />
  <param name="camera" value="$(arg camera)"/>
  <!-- <node pkg="visual_servoing" type="controller.py" name="robot_controller" output="screen"></node> -->
  <node pkg="visual_servoing" type="estimate_pose.py" name="estimate_pose" output="screen"></node>

</launch>
